3.10/programs-arangod-general.md

```diff
@@ -40,15 +40,34 @@ parameter pid-file is given, then the server will report an error and exit.
 
 ## Default Language
 
-`--default-language default-language`
+`--default-language language-name`
 
-The default language ist used for sorting and comparing strings. The language
+The default language is used for sorting and comparing strings. The language
 value is a two-letter language code (ISO-639) or it is composed by a two-letter
-language code with and a two letter country code (ISO-3166). Valid languages are
-"de", "en", "en_US" or "en_UK".
+language code followed by a two letter country code (ISO-3166). For example:
+"de", "en", "en_US", "en_UK".
 
 The default default-language is set to be the system locale on that platform.
 
+## ICU Language
+
+<small>Introduced in: v3.9.1</small>
```

Comment:
Probably not needed here once mentioned in 3.9 docs.

Reply:
It's useful at the very least because it's not in the .0 release (covered by the release notes). One could argue that the manually written startup option pages don't require version hints because we have this information in the generated startup option tables but I'm in favor of having it here, too, for convenience.

Ideally, there would only be a single place instead of two for every startup option information but that requires a design solution first and likely a change to the tooling second.